app/src/main/res/drawable/thumb_werknemerloket.xml

```diff
@@ -0,0 +1,36 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:aapt="http://schemas.android.com/aapt"
+    android:width="106.6dp"
+    android:height="106.6dp"
+    android:viewportWidth="106.6"
+    android:viewportHeight="106.6">
+  <path
+      android:pathData="M0,0h106.6v106.6h-106.6z">
+    <aapt:attr name="android:fillColor">
+      <gradient 
+          android:startY="106.6"
+          android:startX="53.3"
+          android:endY="0"
+          android:endX="53.3"
+          android:type="linear">
+        <item android:offset="0" android:color="#FF41A737"/>
+        <item android:offset="1" android:color="#FFC4DA83"/>
```

Comment:
The specification of `linear` gradient is redundant as it's the default.
And the colours can be reduced from RGBA to RGB by removing the leading two `FF`.